db/migrate/20160720172751_add_at_school_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddAtSchoolToUser < ActiveRecord::Migration
+  def change
+    add_column :users, :at_school, :boolean
```

Comment:
it seems like this might not be awesome at some point down the road, but I think we can worry about it at that point. probably a reasonable solution for now.